models/bert/__main__.py

```diff
@@ -136,5 +136,4 @@ def evaluate_split(model, processor, tokenizer, args, split='dev'):
         model = model.to(device)
 
     evaluate_split(model, processor, tokenizer, args, split='dev')
-    evaluate_split(model, processor, tokenizer, args, split='test')
-
+    evaluate_split(model, processor, tokenizer, args, split='test')
```

Comment:
Can you please remove this change? It's affecting an unrelated file. Also, as a convention, we have newlines at the end of files.